src/scaffold.js

```diff
@@ -60,11 +76,11 @@ dataSources:
       abi: Contract
     mapping:
       kind: ethereum/events
-      apiVersion: 0.0.1
+      apiVersion: 0.0.2
```

Comment:
Since we're doing this big PR to fix `graph init` issue before the release, should we also update the `'@graphprotocol/graph-ts'` version in this file?